docs/editor/voice.md

```diff
@@ -0,0 +1,46 @@
+---
+Order:
+Area: editor
+TOCTitle: Accessibility
+ContentId:
+PageTitle: Using Voice in Visual Studio Code
+DateApproved:
+MetaDescription: Visual Studio Code voice accessibility features. Learn here about the various ways VS Code can be used with voice.
+---
+# Voice Support
+
+Microsoft publishes the [VS Code Speech](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech) extension to enable various features through the power of just using your voice. Once installed, it allows you to dictate into the editor, or use your voice to talk directly into the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) feature.
+
+![VS Code Speech Extension](images/accessibility/speech-extension.png)
+
+## Editor Dictation
+
+You can dictate into the editor via these commands: **Voice: Start Dictation in Editor** (`kb(workbench.action.editorDictation.start)`) and **Voice: Stop Dictation in Editor** (`kb(workbench.action.editorDictation.stop)`). Once started, a little microphone icon appears where the cursor is, awaiting your voice input:
+
+![Editor Dictation Mode](images/accessibility/editor-dictate.png)
+
+You can press and hold the keybinding for the start command (`kb(workbench.action.editorDictation.start)`) to enable walky-talky mode, where the voice recognition stops as soon as you release the keys.
+
+**Note:** dictation even works in other places where a rich editor is used, such as the SCM commit input box and the comments input field when reviewing pull requests.
+
+## Voice in GitHub Copilot Chat
+
+You can use your voice to talk to [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) easily. The command **Voice: Start Voice Chat** (`kb(workbench.action.chat.startVoiceChat)`) works everywhere and brings up a voice chat, depending on where your focus is (inline chat in the editor, panel chat otherwise).
```

Comment:
```suggestion
You can use your voice to talk to [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) easily. The command **Voice: Start Voice Chat** (`kb(workbench.action.chat.startVoiceChat)`) brings up a voice chat, regardless of where the current focus is. If the focus is in the editor, inline chat is started, otherwise the Chat view is opened.
```